pkg/pillar/cmd/baseosmgr/handledownload.go

```diff
@@ -436,6 +437,7 @@ func installDownloadedObject(objType string, safename string,
 		log.Infof("installDownloadedObject(%s) done\n", safename)
 	} else {
 		status.Error = fmt.Sprintf("%s", ret)
+		status.ErrorSource = pubsub.TypeToName(types.VerifyImageStatus{})
```

Comment:
Wondering if it makes sense to add a status.SetError ()
   ->This way, it is easier to enforce setting time / source and error string..

Reply:
done